{{cookiecutter.project_slug}}/k8s/base/django.configmap.yaml

```
@@ -27,3 +27,6 @@ data:
   AWS_S3_ACCESS_KEY_ID: CHANGEME_S3_IAM_USER
   DJANGO_AWS_S3_REGION_NAME: {{cookiecutter.aws_region}}
   DJANGO_AWS_STORAGE_BUCKET_NAME: "CHANGEME_S3_BUCKET_NAME"
+
+  # sentry monitoring
+  SENTRY_DSN: "CHANGEME_SENTRY_KEY"
```

I really don't like CHANGEME values ... They often don't get changed.
```suggestion
  SENTRY_DSN: ""
```